scripts/setup.py

```diff
@@ -78,13 +78,62 @@ def download_and_install_package(url_to_retrieve, filename):
             downloaded.
         filename: string. The name of the tar file.
     """
+
     python_utils.url_retrieve(url_to_retrieve, filename=filename)
     tar = tarfile.open(name=filename)
     tar.extractall(path=common.OPPIA_TOOLS_DIR)
     tar.close()
     os.remove(filename)
 
 
+def download_and_install_node_on_windows(url_to_retrieve, filename):
+    """Downloads and installs node in Oppia tools directory for Windows.
+
+    Args:
+        url_to_retrieve: string. The url to retrieve node.
```

Comment:
The typeinfo should be whatever you get when you call `type(...)`. In this case I think it's either going to be str or unicode (you'll need to check).

Reply:
I think you missed replying to this...

Reply:
This function is removed in the new commit.